Bottom: 2ef7ac15c50cef7d744c10ce8961e1ce32d7f3ba
Top:    ae4323c1f00acb75dee22864a982acc7dd1b1b0c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-25 12:51:54 +0000

Copy contour is_round information when copying polygons


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 7c372c0..296841f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3518,6 +3518,10 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
   (*dst)->xmin = src->xmin, (*dst)->xmax = src->xmax;
   (*dst)->ymin = src->ymin, (*dst)->ymax = src->ymax;
   (*dst)->area = src->area;
+  (*dst)->is_round = src->is_round;
+  (*dst)->cx = src->cx;
+  (*dst)->cy = src->cy;
+  (*dst)->radius = src->radius;
 
   for (cur = NEXT_EDGE (&src->head); cur != &src->head; cur = NEXT_VERTEX (cur))
     {
